components/xPoints/index.tsx

```diff
@@ -60,7 +76,7 @@ function Component() {
                 })
             })
         // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [])
+    }, [dashboardState])
```

Comment:
can you explain the use of dashboardState here @DevPH-Wistkey?

Reply:
Yes, this condition for triggering when user access xpoints before connected to zilpay, so that when zilpay connected(dashboard state changed) useeffect will fetching again. @tralkan